sitedata/README.md

```diff
@@ -53,10 +57,10 @@ The list of papers.
 
 - UID: `<Unique ID>`
 - title: `<paper title>`
-- authors: `<list of authors>` -- (separated by `|` in CSV)
+- authors: `<list of authors>` -- (seperated by `|` in CSV)
```

Comment:
typo in "seperated" - is this being pulled from upstream? If so, we could fix the typo in the upstream repo.

Reply:
Yes, this was inherited from the upstream. Fixed it here. Perhaps we can make a separate consolidated PR for fixes to the upstream. I added some null checks and also removed some duplicate script requests and moved them to the bottom of the page for performance enhancement.

A lot of these minor beneficial changes can be submitted to the upstream.